### DISCUSSION TITLE
[ERROR GRT-0057] Missing track structure for layer metal1

### DISCUSSION THREAD

**Discussion by imcamino** [The-OpenROAD-Project/OpenROAD-flow-scripts#945] [General]
[ERROR GRT-0057] Missing track structure for layer metal1, how to fix?  why the innovus does not exit flow when the log occurs??
![image](https://user-images.githubusercontent.com/122084694/229478048-866e4be4-b634-42ea-804c-2169326be5e8.png)

**Comment by vijayank88** [comment]
@imcamino Which PDK are you using?

**Reply by imcamino** [reply]
i only load the lef&def after placement. the lef is ISPD2015 benchmark case。
![image](https://user-images.githubusercontent.com/122084694/229496866-e2cb456b-66a3-4493-9c39-0e1f86ec7633.png)

**Reply by vijayank88** [reply]
I think for better understanding, provide the steps you executed in script.tcl and required input files.

**Reply by imcamino** [reply]
Thanks your reply . the script & input files plz download from attachment.
[input_error_GRT-0057.zip](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/files/11148578/input_error_GRT-0057.zip)

**Comment by maliberty** [comment]
Why are you mentioning innovus but showing an OpenROAD output?

**Comment by maliberty** [comment]
My first guess is that you need to run 'make_tracks'

**Reply by imcamino** [reply]
Thanks your advise, I will try again after make_tracks。by the way what  value of pitch&offset should i set?

**Reply by vijayank88** [reply]
refer this file for `make_tracks.tcl` https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/blob/master/docs/contrib/PlatformBringUp.md#metal-tracks-configuration and called during floorplan stage.
So you have to start from scratch again.
